sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -6,8 +6,8 @@
         class="mb-2"
         :type="BannerTypes.Warning"
         header="Warning"
-        :summary="warningText.join('\n')"
-      />
+        :summaryList="warningText"
```

Comment:
I think it makes more sense to call it a `warningList`

Reply:
Updated